docs/solutions/sql_server_always_on_availability_groups/terraform.tfvars

```diff
@@ -0,0 +1,18 @@
+tenancy_id = "<tenancy OCID>"
+
+user_id = "<user OCID>"
```

Comment:
All of our other examples are using "user_ocid".

Reply:
Ditto